vms/platformvm/txs/camino_claim_tx.go

```diff
@@ -10,51 +10,42 @@ import (
 	"github.com/ava-labs/avalanchego/ids"
 	"github.com/ava-labs/avalanchego/snow"
 	"github.com/ava-labs/avalanchego/utils/set"
-	"github.com/ava-labs/avalanchego/vms/platformvm/fx"
 	"github.com/ava-labs/avalanchego/vms/platformvm/locked"
 )
 
 var (
 	_ UnsignedTx = (*ClaimTx)(nil)
 
-	errNoDepositsOrClaimables = errors.New("no deposit txs with rewards or claimables to claim")
-	errNonUniqueDepositTxID   = errors.New("non-unique deposit tx id")
-	errNonUniqueOwnerID       = errors.New("non-unique owner id")
-	errWrongClaimedAmount     = errors.New("zero claimed amount or amounts len doesn't match ownerIDs len")
-	errWrongClaimType         = errors.New("wrong claim type")
+	errNoClaimables         = errors.New("no deposit txs with rewards or claimables to claim")
+	errNonUniqueClaimableID = errors.New("non-unique deposit tx id or claimable owner id")
+	errZeroClaimedAmount    = errors.New("zero claimed amount")
+	ErrWrongClaimType       = errors.New("wrong claim type")
 
-	ClaimTypeValidatorReward      ClaimType = 0b01
-	ClaimTypeExpiredDepositReward ClaimType = 0b10
-	ClaimTypeAll                  ClaimType = 0b11
+	ClaimTypeValidatorReward      ClaimType = 0b001
```

Comment:
IMO we don't have a bit field anymore, it should be an enum now (??) and "AllTreasury" can be removed"